modules/connectors/hay_post/karrio/providers/hay_post/error.py

```diff
@@ -0,0 +1,29 @@
+import typing
+import karrio.lib as lib
+import karrio.core.models as models
+import karrio.providers.hay_post.utils as provider_utils
+
+import karrio.schemas.hay_post.error as hay_post
+
+
+def parse_error_response(
+    response: dict,
+    settings: provider_utils.Settings,
+    **kwargs,
+) -> typing.List[models.Message]:
+    responses = response if isinstance(response, list) else [response]
+
+    errors: typing.List[hay_post.ErrorType] = [
+        lib.to_object(hay_post.ErrorType, error) for error in responses
+    ]
+
+    return [
+        models.Message(
+            carrier_id=settings.carrier_id,
+            carrier_name=settings.carrier_name,
+            code="404",
+            message=" ".join(error.key.split("_")).capitalize(),
```

Comment:
Is there a reason why the error needs to be formatted without underscores and capitalized?

Reply:
To make it more readable, I changed the error message from {"key": "expired_access_token"} to "Expired Access Token" since HayPost returns errors in the former format.

Reply:
To make it more readable, I changed the error message from {"key": "expired_access_token"} to "Expired Access Token" since HayPost returns errors in the former format.